Examples/Tests/PML/analysis_pml_psatd.py

```diff
@@ -20,7 +19,7 @@
 ############################
 ### INITIAL LASER ENERGY ###
 ############################
-energy_start = 9.1301289517e-08
+energy_start = 7.297301362346945e-08 # electromagnetic energy at iteration 50
```

Comment:
Maybe we could also compute the starting energy from the diags and assert that the value is not too different from the one here?

This would pretty much be another checksum, but in case the checksums are failing this one would indicate if the difference is coming from the antenna+solver or from the PML.

Reply:
@NeilZaim Thank you for your suggestion. I implemented the change in https://github.com/ECP-WarpX/WarpX/pull/1513/commits/e231246048ea92b482ff463b8154384f45df9f83, I hope this is what you had in mind.